src/registrar/assets/sass/_theme/_buttons.scss

```diff
@@ -94,10 +120,22 @@ a.usa-button--outline:active {
   color: color('primary-darker');
 }
 
+// Used on: Domain request withdraw confirmation
 a.withdraw {
   background-color: color('error');
 }
+
+a.withdraw:hover,
+a.withdraw:focus {
+  background-color: color('error-dark');
+}
   
+a.withdraw:active {
+  background-color: color('error-darker');
+}
+
+// Used on: Domain request status
+//NOTE: Revise to BEM convention usa-button--outline-secondary
```

Comment:
Save me referencing BEM, if this isn't up to the convention, note it in a refactor ticket?